corehq/apps/reports/tests/test_tasks.py

```diff
@@ -1,37 +1,70 @@
 from django.test import SimpleTestCase
 
-from corehq.apps.reports.tasks import find_question_id
+from corehq.apps.reports.tasks import _get_question_id_for_attachment, _find_path_to_question_id
 
 
-class FindQuestionIdTests(SimpleTestCase):
+class GetQuestionIdTests(SimpleTestCase):
```

Comment:
I like that you wrote a separate test suite for this. I'm conflicted on whether or not `FindQuestionIdTests` are necessary anymore -- it seems like FindQuestionId is just a helper method, and the behavior we ultimately care about is in `_get_question_id_for_attachment`. Either way, I think an explicit regression test would be nice to see for this bug: if an absolute path is specified but not found, that it returns the filename, if it exists.